src/App/Formatter/Post/CSV.php

```diff
@@ -379,6 +354,39 @@ private function getValueFromRecord($record, $keyParam, $attributes)
         return $return;
     }
 
+    private function writeStreamToFS($stream)
```

Comment:
@webong, just double checking, did anything really change with this method? Or is it just that the differ lost context as the placement changed inside the file? It looks pretty much the same to me...

Reply:
Just the placement of the method changed.